src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Marshaling/JSMarshalerArgument.Func.cs

```diff
@@ -457,6 +513,9 @@ public unsafe void ToJS<T>(Action<T> value, ArgumentToManagedCallback<T> arg1Mar
         /// Implementation of the argument marshaling.
         /// It's used by JSImport code generator and should not be used by developers in source code.
         /// </summary>
+        /// <typeparam name="TResult">Type of marshaled result of the Func</typeparam>
+        /// <param name="value">Value to be marshaled.</param>
+        /// <param name="resMarshaler">Generated callback which marshals the result of the Func</param>
```

Comment:
Suggestion applies to all similar texts:
```suggestion
        /// <param name="resMarshaler">The generated callback which marshals the result of the function.</param>
```